src/components/backend-ai-login.ts

```diff
@@ -110,6 +110,7 @@ export default class BackendAILogin extends BackendAIPage {
   @property({type: Boolean}) maxShmPerContainer = 2;
   @property({type: Boolean}) maxFileUploadSize = -1;
   @property({type: Boolean}) maskUserInfo = false;
+  @property({type: Boolean}) hideAgents = false;
```

Comment:
In https://github.com/lablup/backend.ai-webui/pull/1593/files#diff-efcecd6c9b6f3d9e866667b37775627d072d9cde0d40326edcba878e773fc109R712, the default value is true but this is false. Wouldn't it be better to unify?

Reply:
Yup, you're right I'll change it.